pkgs/tools/typesetting/tex/texlive/fixedHashes.nix

```diff
@@ -1,9095 +1 @@
-{
```

Comment:
@veprbl I didn't notice `fixedHashes.nix` was empty... is this intentional? (I am generating the April one right now, by the way.)

Reply:
Must be a bug. A fix would be appreciated.